packages/discovery-provider/src/queries/download_csv.py

```diff
@@ -185,57 +192,85 @@ def download_purchases(args: DownloadPurchasesArgs):
     return to_download
 
 
+def format_sale_for_csv(result, seller_handle, seller_user_id, include_email=False):
+    """Format a sale result into a CSV-friendly dictionary format."""
+    # Convert datetime to ISO format string
+    created_at = result.created_at.isoformat() if result.created_at else None
```

Comment:
needed to convert for JSON serialization sakes. seems it was already a string version of an iso string though